en/02_Developer_Guides/03_Forms/00_Introduction.md

```diff
@@ -22,12 +18,12 @@ Creating a [Form](api:SilverStripe\Forms\Form) has the following signature.
 use SilverStripe\Forms\Form;
 use SilverStripe\Forms\FieldList;
 
-$form = new Form(
+$form = Form::create(
     $controller, // the Controller to render this form on 
     $name, // name of the method that returns this form on the controller
-    FieldList $fields, // list of FormField instances 
-    FieldList $actions, // list of FormAction instances
-    $required // optional use of RequiredFields object
+    $fields, // list of FormField instances 
+    $actions, // list of FormAction instances
+    $validator // optional Validator - usually a RequiredFields object
```

Comment:
Should we mention the CompositeValidator here instead to make people more aware of it? Not sure on the exact wording though.

Reply:
I thought about that too but I couldn't think of a good way to discuss it, especially since other than the composite validator the only built-in validator is requiredfields anyway.

Ultimately decided it's more effort than it's worth and tech ically out of scope for this PR anyway.

Reply:
It's mentioned further down in other places, so we can probably leave it out here.